decentraland/unity-explorer
# style: update emotes textures

## What does this PR change?
As the emotes icon, represented by a hand, was not very intuitive there is a new representation of a humanoid dancing as an icon.

## How to test the changes?
This new icon impacted the backpack emotes section, the emotes persistent button, and the emotes slots from the wheel as marked in the images bellow.
<img width="809" alt="Screenshot 2024-07-18 at 12 52 02" src="https://github.com/user-attachments/assets/2867eb56-026d-41ec-bca9-2dc46996d20c">
<img width="752" alt="Screenshot 2024-07-18 at 12 51 54" src="https://github.com/user-attachments/assets/17b3a0ff-1cd7-48e4-a339-a6b418e15d72">

1. Launch the explorer
2. Check the HUD emotes icon on and off states are okay. 
3. Open the emotes wheel pressing `B` or by clicking the emotes button. Check the icon is updated on the slots' flaps.
4. While the wheel is opened, press `E` or `I` and check the emotes section icon has been updated. Same with emote items flap.
